cmd/marker/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM centos:centos7
```

Comment:
maybe it will be good to have a multi stage build here?

Reply:
Was this comment addressed? Doesn't seem like it. (And when I say addressed, I mean answered.)